src/components/Glossary/Glossary.tsx

```diff
@@ -0,0 +1,12 @@
+import classNames from 'classnames';
+
+import { Table } from '@/components/ui/table';
+
+interface GlossaryProps {
```

Comment:
just learned this today, you can do
```suggestion
interface GlossaryProps extends React.PropsWithChildren {
  className?: string;
}
```

and not have to add `children` as a prop.